src/elasticarray.jl

```diff
@@ -19,7 +19,7 @@ Constructors:
     ElasticArray{T}(dims::Integer...)
     convert(ElasticArray, A::AbstractArray)
 """
-struct ElasticArray{T,N,M,V<:DenseVector{T}} <: DenseArray{T,N}
+struct ElasticArray{T,N,M,V <: DenseVector{T}} <: DenseArray{T,N}
```

Comment:
Could you maybe remove these whitespace changes?

Reply:
Sure. It seems that VS code formats it automatically though. I am not familiar with the PR workflow. Shall I propose a new PR when proper changes have been made?

Reply:
Normally, you can add more commits, but not if you did a PR based on your master branch. In general, the recommendation is to always create a dedicated branch on your clone of the repo for the PR (see https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests).

It's probably best if you close this PR, create a branch, and create a new one.